### DISCUSSION TITLE
Risk Graph question

### DISCUSSION THREAD

**Discussion by mepage0113** [Profitbase/PowerBI-visuals-Risk#2] [Q&A]
Hi,
I am using the free version of the Profitbase Risk Graph visual, and I have run into an issue. The settings on the axis max range for the x axis and the y axis in the range options change when I update the selection in my slicers. Both settings are defaulted to 4 when I first setup the visual. Everything looks fine in the graph, until I change my slicer values and the settings go to 16 for the x and 15 for the y. Then the bubbles show up in the wrong box. I can change the max range back in edit mode, but as soon as I change the slicer values again, back they go to 16 and 15. I want those max settings to stay at 4 for both the x and y axis. What causes those settings to change when the data selected for the visual changes?

**Comment by AndersAano** [comment]
Hi @mepage0113,

Sorry for the delayed response. We'll look into this asap.

Best regards
Anders

**Reply by AndersAano** [reply]
Hi, this should have been fixed by now.

-Anders
